platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/JewelFlags.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.jewel.foundation
+
+/**
+ * JewelFlags is an object that holds configuration flags used in the Jewel library.
+ *
+ * These flags can control specific behaviors or enable experimental features within Jewel.
+ */
+public object JewelFlags {
+
+    /**
+     * If true, the Jewel library will use a custom popup renderer. This is an experimental feature and may not be fully
+     * stable.
+     *
+     * To enable this feature, you can either set the system property `jewel.customPopupRender` to `true`, or pass the
+     * `-Djewel.customPopupRender=true` argument when running your application.
+     *
+     * Note that this flag affects both popups, menus and tooltips rendering from Jewel Components.
+     */
```

Comment:
I reckon we also need to explain what this does and how it interacts with the Compose layerType property:

```suggestion
    /**
     * Enable custom popups handling in Jewel.
     * The default value is `false`.
     *
     * If enabled, the Jewel library will use a custom popup renderer, using separate 
     * windows instead of being drawn onto the same layer. 
     * 
     * This is an experimental feature and may not be fully stable. When enabled, Compose's popup
     * settings are ignored when using Jewel popups and tooltips. This means that setting
     * `compose.layers.type` will have no effect on Jewel popups and tooltips.
     *
     * To set this flag, you can also set the system property `jewel.customPopupRender` to `true`/`false`,
     * or pass the `-Djewel.customPopupRender=[true|false]` argument when
     * running your application.
     *
     * Note that this flag affects popups, menus and tooltips rendering from Jewel Components.
     * It does not affect `Dialog`s.
     */
```